### ISSUE TITLE
AttributeError: 'module' object has no attribute 'add'

### ISSUE THREAD

**Issue by lordcoste** [jezdez/Flask-RQ2#11]
Hello, I'm getting this error on the worker when enqueue this simple job:

```python
from flask import Flask
from flask_rq2 import RQ

app = Flask(__name__)

rq = RQ(app)

@rq.job
def add(x, y):
    return x + y

@app.route('/test')
def my_test():
    job = add.queue(1, 2)
    return 'ok'
```

```
19:14:43 default: __main__.add(1, 2) (75ab9148-4aca-44bc-a41e-cffd368509ba)
19:14:43 AttributeError: 'module' object has no attribute 'add'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/rq/worker.py", line 700, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python2.7/site-packages/rq/job.py", line 500, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/usr/local/lib/python2.7/site-packages/rq/job.py", line 206, in func
    return import_attribute(self.func_name)
  File "/usr/local/lib/python2.7/site-packages/rq/utils.py", line 151, in import_attribute
    return getattr(module, attribute)
AttributeError: 'module' object has no attribute 'add'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/rq/worker.py", line 700, in perform_job
    rv = job.perform()
  File "/usr/local/lib/python2.7/site-packages/rq/job.py", line 500, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/usr/local/lib/python2.7/site-packages/rq/job.py", line 206, in func
    return import_attribute(self.func_name)
  File "/usr/local/lib/python2.7/site-packages/rq/utils.py", line 151, in import_attribute
    return getattr(module, attribute)
AttributeError: 'module' object has no attribute 'add'
19:14:43 Moving job to u'failed' queue
```

I'm running everything inside this docker container:
```
FROM tiangolo/uwsgi-nginx-flask:flask-upload

RUN pip install rq
RUN pip install Flask-RQ2
```

Hope you can help me.

**Comment by Xanagandr** [comment]
I'm having the same issue. Here's the code for my worker:

`import redis
from rq import Worker, Queue, Connection

listen = ['default']

redis_url = 'redis://localhost:6379'

conn = redis.from_url(redis_url)

if __name__ == '__main__':
    with Connection(conn):
        worker = Worker(list(map(Queue, listen)))
        worker.work()
`

**Comment by jezdez** [comment]
@lordcoste I'm afraid this is an issue I've only just now was able to fix, and sadly by doing a big refactor in #41. Make sure you set `FLASK_APP` correctly, or the RQ worker won't be able to locate the job correctly. This works for me now.

@Xanagandr Please don't write your own worker, but use the provider worker command `flask rq worker`.
